spec/amp-story-player.md

```diff
@@ -126,3 +130,192 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player. These methods are exposed on the DOM HTML element, `const playerEl = document.querySelector('amp-story-player')` and on the global class variable,`const player = new AmpStoryPlayer(window, playerEl)`.
```

Comment:
Just a suggestion: should we document only one of these to be more clear? In both cases you have a reference to `playerEl`.

Two nits:
- the `DOM` word does not add any information and can be dropped
- in your example you say the methods are exposed on the global class variable which would be `AmpStoryPlayer`, but your example shows that the methods are on the instance of that class